gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/Channelizer.java

```diff
@@ -18,6 +18,7 @@
  */
 package org.apache.tinkerpop.gremlin.driver;
 
+import io.netty.handler.ssl.SslHandler;
```

Comment:
very minor: this import should be in line 37

Reply:
Not minor, actually a requirement for properly ordered imports

Reply:
I'll force push a change to update this.